### ISSUE TITLE
PPOCRV3模型断点训练的问题。

### ISSUE THREAD

**Issue by ainndejj11** [PaddlePaddle/PaddleOCR#6509]
问题：
训练PPOCRV3的文字识别模型时，模型断点训练时，损失和学习率是接着latest模型训练，但是epoch却是从1开始。
用ch_ppocr_v2.0就没这个问题

命令：
python tools/train.py -c {配置文件} -o Global.checkpoints=output/output/my_rec_PPocr_v3_shouxie/latest.pdparams

训练日志：
[2022/06/07 02:14:41] ppocr INFO: epoch: [21/500], global_step: 88760, lr: 0.000998, acc: 0.324074, norm_edit_dis: 0.830847, CTCLoss: 6.884216, SARLoss: 1.068864, loss: 7.993740, avg_reader_cost: 0.03827 s, avg_batch_cost: 0.21293 s, avg_samples: 27.0, ips: 126.79935 samples/s, eta: 8 days, 5:57:28
[2022/06/07 09:09:38] ppocr INFO: Architecture : 
[2022/06/07 09:09:38] ppocr INFO:     Backbone : 
[2022/06/07 09:09:38] ppocr INFO:         last_conv_stride : [1, 2]
[2022/06/07 09:09:38] ppocr INFO:         last_pool_type : avg
[2022/06/07 09:09:38] ppocr INFO:         name : MobileNetV1Enhance
[2022/06/07 09:09:38] ppocr INFO:         scale : 0.5
[2022/06/07 09:09:38] ppocr INFO:     Head : 
[2022/06/07 09:09:38] ppocr INFO:         head_list : 
[2022/06/07 09:09:38] ppocr INFO:             CTCHead : 
[2022/06/07 09:09:38] ppocr INFO:                 Head : 
[2022/06/07 09:09:38] ppocr INFO:                     fc_decay : 1e-05
[2022/06/07 09:09:38] ppocr INFO:                 Neck : 
[2022/06/07 09:09:38] ppocr INFO:                     depth : 2
[2022/06/07 09:09:38] ppocr INFO:                     dims : 64
[2022/06/07 09:09:38] ppocr INFO:                     hidden_dims : 120
[2022/06/07 09:09:38] ppocr INFO:                     name : svtr
[2022/06/07 09:09:38] ppocr INFO:                     use_guide : True
[2022/06/07 09:09:38] ppocr INFO:             SARHead : 
[2022/06/07 09:09:38] ppocr INFO:                 enc_dim : 512
[2022/06/07 09:09:38] ppocr INFO:                 max_text_length : 25
[2022/06/07 09:09:38] ppocr INFO:         name : MultiHead
[2022/06/07 09:09:38] ppocr INFO:     Transform : None
[2022/06/07 09:09:38] ppocr INFO:     algorithm : SVTR
[2022/06/07 09:09:38] ppocr INFO:     model_type : rec
[2022/06/07 09:09:38] ppocr INFO: Eval : 
[2022/06/07 09:09:38] ppocr INFO:     dataset : 
[2022/06/07 09:09:38] ppocr INFO:         data_dir : dataset_shouxie/
[2022/06/07 09:09:38] ppocr INFO:         label_file_list : ['dataset_shouxie/test.txt']
[2022/06/07 09:09:38] ppocr INFO:         name : SimpleDataSet
[2022/06/07 09:09:38] ppocr INFO:         transforms : 
[2022/06/07 09:09:38] ppocr INFO:             DecodeImage : 
[2022/06/07 09:09:38] ppocr INFO:                 channel_first : False
[2022/06/07 09:09:38] ppocr INFO:                 img_mode : BGR
[2022/06/07 09:09:38] ppocr INFO:             MultiLabelEncode : None
[2022/06/07 09:09:38] ppocr INFO:             RecResizeImg : 
[2022/06/07 09:09:38] ppocr INFO:                 image_shape : [3, 48, 320]
[2022/06/07 09:09:38] ppocr INFO:             KeepKeys : 
[2022/06/07 09:09:38] ppocr INFO:                 keep_keys : ['image', 'label_ctc', 'label_sar', 'length', 'valid_ratio']
[2022/06/07 09:09:38] ppocr INFO:     loader : 
[2022/06/07 09:09:38] ppocr INFO:         batch_size_per_card : 54
[2022/06/07 09:09:38] ppocr INFO:         drop_last : False
[2022/06/07 09:09:38] ppocr INFO:         num_workers : 4
[2022/06/07 09:09:38] ppocr INFO:         shuffle : False
[2022/06/07 09:09:38] ppocr INFO: Global : 
[2022/06/07 09:09:38] ppocr INFO:     cal_metric_during_train : True
[2022/06/07 09:09:38] ppocr INFO:     character_dict_path : ppocr/utils/ppocr_keys_v1.txt
[2022/06/07 09:09:38] ppocr INFO:     checkpoints : output/my_rec_PPocr_v3_shouxie/latest.pdparams
[2022/06/07 09:09:38] ppocr INFO:     debug : False
[2022/06/07 09:09:38] ppocr INFO:     distributed : False
[2022/06/07 09:09:38] ppocr INFO:     epoch_num : 500
[2022/06/07 09:09:38] ppocr INFO:     eval_batch_step : 4000
[2022/06/07 09:09:38] ppocr INFO:     infer_img : doc/imgs_words/ch/word_1.jpg
[2022/06/07 09:09:38] ppocr INFO:     infer_mode : False
[2022/06/07 09:09:38] ppocr INFO:     log_smooth_window : 20
[2022/06/07 09:09:38] ppocr INFO:     max_text_length : 25
[2022/06/07 09:09:38] ppocr INFO:     pretrained_model : my_pretrain/ch_PP-OCRv3_rec_train/best_accuracy
[2022/06/07 09:09:38] ppocr INFO:     print_batch_step : 40
[2022/06/07 09:09:38] ppocr INFO:     save_epoch_step : 3
[2022/06/07 09:09:38] ppocr INFO:     save_inference_dir : None
[2022/06/07 09:09:38] ppocr INFO:     save_model_dir : ./output/my_rec_PPocr_v3_shouxie
[2022/06/07 09:09:38] ppocr INFO:     save_res_path : ./output/my_rec_PPocr_v3_shouxie/predicts_ppocrv3.txt
[2022/06/07 09:09:38] ppocr INFO:     use_gpu : True
[2022/06/07 09:09:38] ppocr INFO:     use_space_char : True
[2022/06/07 09:09:38] ppocr INFO:     use_visualdl : False
[2022/06/07 09:09:38] ppocr INFO: Loss : 
[2022/06/07 09:09:38] ppocr INFO:     loss_config_list : 
[2022/06/07 09:09:38] ppocr INFO:         CTCLoss : None
[2022/06/07 09:09:38] ppocr INFO:         SARLoss : None
[2022/06/07 09:09:38] ppocr INFO:     name : MultiLoss
[2022/06/07 09:09:38] ppocr INFO: Metric : 
[2022/06/07 09:09:38] ppocr INFO:     ignore_space : False
[2022/06/07 09:09:38] ppocr INFO:     main_indicator : acc
[2022/06/07 09:09:38] ppocr INFO:     name : RecMetric
[2022/06/07 09:09:38] ppocr INFO: Optimizer : 
[2022/06/07 09:09:38] ppocr INFO:     beta1 : 0.9
[2022/06/07 09:09:38] ppocr INFO:     beta2 : 0.999
[2022/06/07 09:09:38] ppocr INFO:     lr : 
[2022/06/07 09:09:38] ppocr INFO:         learning_rate : 0.001
[2022/06/07 09:09:38] ppocr INFO:         name : Cosine
[2022/06/07 09:09:38] ppocr INFO:         warmup_epoch : 5
[2022/06/07 09:09:38] ppocr INFO:     name : Adam
[2022/06/07 09:09:38] ppocr INFO:     regularizer : 
[2022/06/07 09:09:38] ppocr INFO:         factor : 3e-05
[2022/06/07 09:09:38] ppocr INFO:         name : L2
[2022/06/07 09:09:38] ppocr INFO: PostProcess : 
[2022/06/07 09:09:38] ppocr INFO:     name : CTCLabelDecode
[2022/06/07 09:09:38] ppocr INFO: Train : 
[2022/06/07 09:09:38] ppocr INFO:     dataset : 
[2022/06/07 09:09:38] ppocr INFO:         data_dir : dataset_shouxie/
[2022/06/07 09:09:38] ppocr INFO:         ext_op_transform_idx : 1
[2022/06/07 09:09:38] ppocr INFO:         label_file_list : ['dataset_shouxie/train.txt']
[2022/06/07 09:09:38] ppocr INFO:         name : SimpleDataSet
[2022/06/07 09:09:38] ppocr INFO:         transforms : 
[2022/06/07 09:09:38] ppocr INFO:             DecodeImage : 
[2022/06/07 09:09:38] ppocr INFO:                 channel_first : False
[2022/06/07 09:09:38] ppocr INFO:                 img_mode : BGR
[2022/06/07 09:09:38] ppocr INFO:             RecConAug : 
[2022/06/07 09:09:38] ppocr INFO:                 ext_data_num : 2
[2022/06/07 09:09:38] ppocr INFO:                 image_shape : [48, 320, 3]
[2022/06/07 09:09:38] ppocr INFO:                 prob : 0.5
[2022/06/07 09:09:38] ppocr INFO:             RecAug : None
[2022/06/07 09:09:38] ppocr INFO:             MultiLabelEncode : None
[2022/06/07 09:09:38] ppocr INFO:             RecResizeImg : 
[2022/06/07 09:09:38] ppocr INFO:                 image_shape : [3, 48, 320]
[2022/06/07 09:09:38] ppocr INFO:             KeepKeys : 
[2022/06/07 09:09:38] ppocr INFO:                 keep_keys : ['image', 'label_ctc', 'label_sar', 'length', 'valid_ratio']
[2022/06/07 09:09:38] ppocr INFO:     loader : 
[2022/06/07 09:09:38] ppocr INFO:         batch_size_per_card : 54
[2022/06/07 09:09:38] ppocr INFO:         drop_last : True
[2022/06/07 09:09:38] ppocr INFO:         num_workers : 4
[2022/06/07 09:09:38] ppocr INFO:         shuffle : True
[2022/06/07 09:09:38] ppocr INFO: profiler_options : None
[2022/06/07 09:09:38] ppocr INFO: train with paddle 2.2.2 and device CUDAPlace(0)
[2022/06/07 09:09:38] ppocr INFO: Initialize indexs of datasets:['dataset_shouxie/train.txt']
[2022/06/07 09:09:38] ppocr INFO: Initialize indexs of datasets:['dataset_shouxie/test.txt']
[2022/06/07 09:10:00] ppocr INFO: resume from output/my_rec_PPocr_v3_shouxie/latest
[2022/06/07 09:10:00] ppocr INFO: train dataloader has 4437 iters
[2022/06/07 09:10:00] ppocr INFO: valid dataloader has 320 iters
[2022/06/07 09:10:15] ppocr INFO: epoch: [1/500], global_step: 40, lr: 0.000998, acc: 0.342593, norm_edit_dis: 0.839076, CTCLoss: 6.154938, SARLoss: 1.062240, loss: 7.240303, avg_reader_cost: 0.03460 s, avg_batch_cost: 0.35631 s, avg_samples: 54.0, ips: 151.55528 samples/s, eta: 9 days, 3:34:09
[2022/06/07 09:10:26] ppocr INFO: epoch: [1/500], global_step: 80, lr: 0.000998, acc: 0.324074, norm_edit_dis: 0.838477, CTCLoss: 6.253406, SARLoss: 1.024214, loss: 7.210162, avg_reader_cost: 0.00010 s, avg_batch_cost: 0.29357 s, avg_samples: 54.0, ips: 183.94232 samples/s, eta: 8 days, 8:14:08
[2022/06/07 09:10:39] ppocr INFO: epoch: [1/500], global_step: 120, lr: 0.000998, acc: 0.287037, norm_edit_dis: 0.817675, CTCLoss: 6.540972, SARLoss: 1.067353, loss: 7.660158, avg_reader_cost: 0.00012 s, avg_batch_cost: 0.30329 s, avg_samples: 54.0, ips: 178.04689 samples/s, eta: 8 days, 3:47:08
[2022/06/07 09:10:52] ppocr INFO: epoch: [1/500], global_step: 160, lr: 0.000998, acc: 0.342593, norm_edit_dis: 0.837754, CTCLoss: 6.213328, SARLoss: 1.038592, loss: 7.296705, avg_reader_cost: 0.00018 s, avg_batch_cost: 0.34560 s, avg_samples: 54.0, ips: 156.25208 samples/s, eta: 8 days, 8:04:34
[2022/06/07 09:11:06] ppocr INFO: epoch: [1/500], global_step: 200, lr: 0.000998, acc: 0.342593, norm_edit_dis: 0.841209, CTCLoss: 5.889498, SARLoss: 1.032500, loss: 6.893699, avg_reader_cost: 0.00019 s, avg_batch_cost: 0.34338 s, avg_samples: 54.0, ips: 157.25982 samples/s, eta: 8 days, 10:22:33
[2022/06/07 09:11:20] ppocr INFO: epoch: [1/500], global_step: 240, lr: 0.000998, acc: 0.379630, norm_edit_dis: 0.843180, CTCLoss: 5.783564, SARLoss: 0.926491, loss: 6.699171, avg_reader_cost: 0.00019 s, avg_batch_cost: 0.33799 s, avg_samples: 54.0, ips: 159.76685 samples/s, eta: 8 days, 11:21:15

**Comment by WenmuZhou** [comment]
目前只有在存best模型的时候才会存epoch信息，我们稍后优化一下
